node_package/src/clientStartup.ts

```diff
@@ -304,11 +304,6 @@ export function clientStartup(context: Context): void {
 
   debugTurbolinks('Adding DOMContentLoaded event to install event listeners.');
 
-  // So  long as the document is not loading, we can assume:
-  // The document has finished loading and the document has been parsed
-  // but sub-resources such as images, stylesheets and frames are still loading.
-  // If lazy asynch loading is used, such as with loadable-components, then the init
-  // function will install some handler that will properly know when to do hyrdation.
```

Comment:
Should we put back in a comment reflecting your research as to why this changed?